Cargo.toml

```diff
@@ -1,15 +1,15 @@
 [package]
 name = "compute-starter-kit-rust-beacon-termination"
 version = "0.2.0"
-authors = ["Dora Militaru <dmilitaru@fastly.com>"]
+authors = []
 edition = "2018"
 
 [profile.release]
 debug = true
```

Comment:
Think we are typically removing these?

Reply:
Really? The default starter kit still has it.